packages/playground/wordpress/src/index.ts

```diff
@@ -49,14 +49,6 @@ export async function setupPlatformLevelMuPlugins(php: UniversalPHP) {
 	await php.writeFile(
 		'/internal/shared/mu-plugins/0-playground.php',
 		`<?php
-        // Redirect /wp-admin to /wp-admin/
```

Comment:
This should no longer be necessary since the request handler now redirects to add a trailing slash for existing dirs.